connector/rest/src/main/java/it/pagopa/selfcare/mscore/connector/rest/mapper/UserMapperClient.java

```diff
@@ -0,0 +1,25 @@
+package it.pagopa.selfcare.mscore.connector.rest.mapper;
+
+import it.pagopa.selfcare.mscore.model.user.User;
+import it.pagopa.selfcare.user_registry.generated.openapi.v1.dto.UserId;
+import it.pagopa.selfcare.user_registry.generated.openapi.v1.dto.UserResource;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
+
+
+
+
+@Mapper(componentModel = "spring")
+public interface UserMapperClient {
+
+    User toUser (UserResource userResource);
+
+    @Mapping(target = "id", source = ".", qualifiedByName = "getId")
```

Comment:
you could use directly @Mapping(target = "id", source = "userId.id") without qualifiedByName